charts/launch-agent/README.md

```diff
@@ -4,33 +4,42 @@ This chart deploys the W&B Launch Agent to your Kubernetes cluster.
-| `serviceAccount.annotations` | object | No | `{}` | Annotations to add to the service account created for the agent.
-| `azureStorageAccessKey` | string | No | "" | Azure storage access key required for kaniko to acces build contexts in azure blob storage.
+```bash
+helm upgrade --namespace=wandb --create-namespace --install wandb-launch wandb/launch-agent -f ./values.yaml --namespace=wandb-launch
+```
+
+## Chart variables
+The table below describes all the available variables in the chart:
+
+| Variable                      | Type            | Required | Default                           | Description                                                                                                                                      |
+| ----------------------------- | --------------  | -------- | --------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------   |
+| `agent.labels`                | object          | No       | {}                                | Labels that will be added to the agent deployment.                                                                                               |
+| `agent.apiKey`                | string          | **Yes**  | ""                                | W&B API key to be used by the agent.                                                                                                             |
+| `agent.image`                 | string          | No       | `wandb/launch-agent-dev:latest`   | Container image for the agent.                                                                                                                   |
+| `agent.imagePullPolicy`       | string          | No       | `Always`                          | Pull policy for the agent container image.                                                                                                       |
+| `agent.resources`             | object          | No       | Limit to 1 CPU, 1Gi RAM           | Pod spec resources block for the agent.                                                                                                          |
+| `namespace`                   | string          | No       | `wandb`                           | The namespace to deploy the agent into.                                                                                                          |
+| `additionalTargetNamespaces`  | list(string)    | No       | [`wandb`,`default`]               | The namespace to deploy the agent into.                                                                                                          |
+| `launchConfig`                | mutiline string | **Yes**  | `https://api.wandb.ai`            | URL of your W&B server.                                                                                                                          |
```

Comment:
I think the vriable here should be baseUrl? And we need another line for launchConfig?

Reply:
Updated